string workspace_name
string package_name
string launchfile_name
string[] args
bool debug
